EdFi.Tools.ApiPublisher.Tests/Extensions/LoggerRepositoryExtensions.cs

```diff
@@ -1,19 +1,18 @@
+using Serilog;
 using System;
 using System.Linq;
-using log4net.Appender;
-using log4net.Repository;
 
 namespace EdFi.Tools.ApiPublisher.Tests.Extensions
 {
     public static class LoggerRepositoryExtensions
     {
-        public static string LoggedContent(this ILoggerRepository loggerRepository)
-        {
-            // Inspect the log entries
-            var memoryAppender = loggerRepository.GetAppenders().OfType<MemoryAppender>().Single();
-            var events = memoryAppender.GetEvents();
+        //public static string LoggedContent(this ILogger loggerRepository)
+        //{
+        //    //// Inspect the log entries
+        //    //var memoryAppender = loggerRepository.GetAppenders().OfType<MemoryAppender>().Single();
+        //    //var events = memoryAppender.GetEvents();
 
-            return string.Join(Environment.NewLine, events.Select(e => e.RenderedMessage));
-        }
+        //    //return string.Join(Environment.NewLine, events.Select(e => e.RenderedMessage));
+        //}
     }
```

Comment:
@jagudelo-gap 
If this is not required anymore. Please remove entire class file.

Reply:
@CSR2017 The class has been removed.